### ISSUE TITLE
Add proper coverage for SearcEntityMixin in test_entity.py

### ISSUE THREAD

**Issue by oshtaier** [SatelliteQE/nailgun#237]
We have already provided coverage for 'common' scenario for ReadEntityMixin, UpdateEntityMixin and CreateEntityMixin, so it will be nice to have something similar for search functionality

**Comment by elyezer** [comment]
:+1:

**Comment by Ichimonji10** [comment]
To be clear, there are unit tests for the `EntitySearchMixin` in module `tests.test_entity_mixins`. I think what's being requested here are additional tests in `tests.test_entities`.

**Comment by oshtaier** [comment]
Right, to have pretty similar approach for all entities. And, of course, that task has low priority
